docs/src/examples/pde_constrained.md

```diff
@@ -137,48 +162,66 @@ end
 
 function d2dx(u,dx)
     """
-    2nd order Central difference for 1st degree derivative
+    2nd order Central difference for 2nd degree derivative
     """
     return [[zero(eltype(u))]; (u[3:end] - 2.0.*u[2:end-1] + u[1:end-2]) ./ (dx^2); [zero(eltype(u))]]
 end
 ```
 
-### Burgers' Equation
+### Burgers' Differential Equation
+
+Next, we setup our desired set of equations in order to define our problem.
 
 ```
-## ODE descrition of the Physics:
+## ODE description of the Physics:
 function burgers(u,p,t)
-    # Model paramters
+    # Model parameters
     a0, a1 = p
     dx,Nx = xtrs #[1.0,3.0,0.125,100]
-    return 2.0*a0 .* u +  a1 .* d2dx(u, dx) #.*ddx(ddx(u, dx) ,dx)
+    return 2.0*a0 .* u +  a1 .* d2dx(u, dx)
 end
 ```
 
 ### Solve and Plot Ground Truth
 
+We then solve and plot our partial differential equation. This is the true solution which we
+will compare to further on.
+
 ```
 # Testing Solver on linear PDE
 prob = ODEProblem(burgers,u0,tspan,p)
-sol = concrete_solve(prob,Tsit5(), dt=dt,saveat=t);
+sol = solve(prob,Tsit5(), dt=dt,saveat=t);
 
 plot(x, sol.u[1], lw=3, label="t0", size=(800,500))
 plot!(x, sol.u[end],lw=3, ls=:dash, label="tMax")
 ```
 
 ### Using Neural Network for Predictions
```

Comment:
Let's not call it a neural network. It's just a forward model for the PDE, or a prediction function.

Reply:
That's fair. Should I modify the 1-layer neural network comment as well?

Reply:
yes